utils/common.py

```diff
@@ -597,3 +597,12 @@ def render_string_for_graphql(text):
     if text == '':
         return None
     return text
+
+
+# Get heirarchy level of a django model.
+def get_hierarchy_level(parent_instance):
+    level = 1
+    while parent_instance.parent:
+        level += 1
+        parent_instance = parent_instance.parent
+    return level
```

Comment:
Let's move this out of common, as this might now work for all cases.
You can define utils.py in assessment_registry instead of just in the same function where this is used.